utils/logging.py

```diff
@@ -0,0 +1,9 @@
+from loguru import logger
+
+
+class Logger:
+    @staticmethod
+    def set_logger():
+        logger.remove()
```

Comment:
why remove is needed here?

Reply:
Don't remember exactly, but it was in our course. I always used this consctruction and you've never said something about it.

Reply:
Oh. I just remembered. We use it to hide logger messages from terminal,